Minor wording fixes on the RELEASE_PROCESS document (#653)
tektoncd/cli: RELEASE_PROCESS.md

- When the release is done in https://github.com/tektoncd/cli/releases, edit the
  release and change it from pre-release as released.

- When it's released, you can build the rpm package according to this
  [README](tekton/rpmbuild/README.md) document. Make sure you have done the PREREQ to be
  able to upload it to the copr repository. This could take some time if copr
  builder is busy.

> - In parallel building the rpm task you can start building the debian package according to this

Reviewer: ```suggestion
- While  rpm task is building the rpm package, you can start building the debian package according to this
```
